app/src/sharedTest/java/org/oppia/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -68,6 +71,18 @@ class ProfileChooserFragmentTest {
       .build()
       .inject(this)
   }
+  @Test
+  fun testProfileChooserFragment_checkSpanCountOnPortrait_spanCountVerifiedSuccessfully() {
+    ActivityScenario.launch(ProfileActivity::class.java).use {
+      onView(withId(R.id.profile_recycler_view)).check(hasGridItemCount(2))
+  }
+  }  @Test
```

Comment:
@Test should come to next line and there should be 1 line gap between these functions

Reply:
Done.